### PR TITLE
Get coordinates from usereventedit.html and store in Event objects.

### PR DESCRIPTION
**Pull request by justness**
Updated servlets, Event class, and usereventedit.html to account for coordinate parameters.
image

### CODE REVIEW

**Comment by justness** [comment]
Next step for the map would be to grab the final centering of the frame (and as a stretch goal, zoom level) for display to the user. It is currently centered on the organizer's automatic geolocation coordinates rather than the map center when saving which may not always be accurate.

File: portfolio/src/main/java/com/google/sps/servlets/Event.java [hunk 1]
```diff
@@ -21,12 +23,14 @@ public Event(String organizer, String eventName, String location, String descrip
...
         this.key = key;
+        this.centerCoord = centerCoord;
+        this.pathCoords = pathCoords;
     }
 
     private String generatePassword(){
         String password = "";
         for (int i = 0; i < 5; i++){
-            char rand = (char)((int)(Math.random() * 97) + 33); 
+            char rand = (char)((int)(Math.random() * 93) + 33);
```
Comment (mamessner) [inline portfolio/src/main/java/com/google/sps/servlets/Event.java:33]: why the change from 97 to 93?

File: portfolio/src/main/java/com/google/sps/servlets/Event.java [hunk 2]
```diff
@@ -11,8 +11,10 @@
     private final Date end;
     private String password;
     private final Key key;
+    private String centerCoord;
+    private String pathCoords;
 
-    public Event(String organizer, String eventName, String location, String description, Date start, Date end, Key key){
+    public Event(String organizer, String eventName, String location, String description, Date start, Date end, Key key, String centerCoord, String pathCoords){
```
Comment (mamessner) [inline portfolio/src/main/java/com/google/sps/servlets/Event.java:17]: Consider making these coordinates something other than strings if you have some time later (I'm good with this for now), like a class that contains lat and long. It feels a little weird to have these be strings, especially the path since that's multiple coordinates

File: portfolio/src/main/java/com/google/sps/servlets/Event.java (hunk 1 above)
Comment (justness) [reply portfolio/src/main/java/com/google/sps/servlets/Event.java:33]: I noticed some chars that were not usual alphanumeric values, and adjusted the number to correct a previous calculation error.
